src/app/static/src/app/store/translations/locales.ts

```diff
@@ -173,6 +175,53 @@ export interface Translations {
 }
 
 const en: Translations = {
+    accessibility: "Accessibility",
+    accessibilityContent: "<h1>Accessibility on NAOMI</h1>\n" +
```

Comment:
Personally, I think for consistency we should go for the first option and use one component that references its strings against a translation file. However, our main locales file is getting a little bloated so it might be worthwhile creating a new file especially for these translations perhaps?